src/OmniSharp.Cake/Services/RequestHandlers/Diagnostics/CodeCheckHandler.cs

```diff
@@ -14,5 +15,15 @@ public class CodeCheckHandler : CakeRequestHandler<CodeCheckRequest, QuickFixRes
             : base(workspace)
         {
         }
+
+        public override Task<QuickFixResponse> HandleCore(CodeCheckRequest request, IRequestHandler<CodeCheckRequest, QuickFixResponse> service)
+        {
+            if (string.IsNullOrEmpty(request.FileName))
+            {
+                return Task.FromResult(new QuickFixResponse());
+            }
+
+            return service.Handle(request);
```

Comment:
Override Task<TResponse> Handle(TRequest request) and call `base.Handle(request)` instead?